pkg/util/paramtable/autoindex_param.go

```diff
@@ -76,6 +77,14 @@ func (p *autoIndexConfig) init(base *BaseTable) {
 	}
 	p.EnableOptimize.Init(base.mgr)
 
+	p.EnableResultLimitCheck = ParamItem{
+		Key:          "autoIndex.resultLimitCheck",
+		Version:      "2.5.0",
+		DefaultValue: "true",
```

Comment:
shall we enable this feature by default?

Reply:
Yes, so that user will not be affected by improperly reduce-topk from autoindex. And try to set it to false if topk insufficient not caused by autoindex (pk duplication, etc)